fixtures/gravity/index.js

```diff
@@ -0,0 +1,105 @@
+// @flow
+import "babel-polyfill"
+
+import { spawnSync } from "child_process"
+import { basename, dirname, extname, join } from "path"
+import { writeFile } from "fs"
+
+import request from "request-promise"
+// request.debug = true
+
+async function authenticate(email: string, password: string): Promise<string> {
+  try {
+    const response = await request({
+      method: "POST",
+      uri: "https://api.artsy.net/oauth2/access_token",
+      qs: {
+        // Eigen OSS keys https://github.com/artsy/eigen/blob/master/Makefile
+        client_id: "e750db60ac506978fc70",
+        client_secret: "3a33d2085cbd1176153f99781bbce7c6",
```

Comment:
@mzikherman Still, though, I’ve been wondering if I should be creating new OSS ones for projects like these (and Emission’s example app). Do you have any thoughts on that?